tests/github/pki_insecure_client.py

```diff
@@ -45,9 +45,21 @@ def install_ca():
         'fx', 'pki', 'install',
         '-p', str(CA_PATH),
         '--password', str(CA_PASSWORD)
-    ], stdin=subprocess.PIPE)
+    ], stdin=subprocess.PIPE, stdout=subprocess.PIPE)
     time.sleep(3)
-    pki_install.communicate('y'.encode('utf-8'))
+    stdout, _ = pki_install.communicate('y'.encode('utf-8'))
+    while 'API rate limit exceeded' in stdout.decode('utf-8'):
+        minutes_to_wait = 5
```

Comment:
Would it be better to move this constant to the top of the script?
Also, is there any meaning in 5 minutes delay, can we make it smaller?